400-claims.md

```diff
@@ -231,24 +231,23 @@ WfK01YbCWioNVGk=
 
 ### How the Claim is Used
 
-The claim is used to inform any CNAB tooling about how to address a particular installation. For example, given the claim record, a package manager that implements CNAB should be able to:
+The claim is used to inform any CNAB tooling about how to address a particular installation. For example, given the claim record, a runtime that implements the CNAB Claims spec should be able to:
 
 - List the _names_ of the installations, given a _bundle name_
 - Given an installation's _name_, return the _bundle info_ that is installed under that name
 - Given an installation _name_ and a _bundle_, generate a _bundle info_.
   - This is accompanied by running the `install` path in the bundle
 - Given an installation's _name_, replace the _bundle info_ with updated _bundle info_, and update the revision with a new ULID, and the modified timestamp with the current time. This is an upgrade operation.
   - This is accompanied by running the `upgrade` path in the bundle
-- Given an installation's name, mark the claim as deleted.
+- Given an installation's name, mark the installation as deleted.
```

Comment:
We might actually want to change `deleted` to `uninstalled`.

Reply:
Agreed 👍